data/moves.ts

```diff
@@ -2204,9 +2204,13 @@ export const Moves: {[moveid: string]: MoveData} = {
 		priority: 0,
 		flags: {contact: 1, protect: 1, mirror: 1, slicing: 1},
 		// critRatio: 2,
-		secondary: {
-			chance: 100,
-			sideCondition: 'spikes',
+		self: {
+			// TODO fix sheer force interaction
```

Comment:
Diamond Storm just puts in a dummy `secondary: {},`.

Reply:
Worked like a charm, thanks! :)